### PR TITLE
Added Logitech Onboard Memory Manager 1.1.6805.0

### PR DESCRIPTION
**Pull request by aesalazar**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97792)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @aesalazar,

This package doesn't appear to run or give any output when launched in a Sandbox, neither from PowerShell nor Windows Explorer.

**Comment by Trenly** [comment]
Looks like this has a dependency on `Logitech.GHUB`
image

**Comment by aesalazar** [comment]
Thanks guys.  Sorry, I thought I tested it properly but I may not have did a proper reset of the Sandbox before doing the final test.  I assume there is nothing to do right now until dependency support is added to winget?

**Comment by stephengillie** [comment]
> Thanks guys. Sorry, I thought I tested it properly but I may not have did a proper reset of the Sandbox before doing the final test. I assume there is nothing to do right now until dependency support is added to winget?

The best solution is to have the installer install the dependency as well, in an unattended way. But this is not really an option in most PRs which are waiting on dependency support.

**Comment by aesalazar** [comment]
FYI, the dependency seems to be on the VC++ Runtime.  I test in a sandbox with:

```winget install Microsoft.VCRedist.2015+.x64```

and it worked like charm.  When support it added, I will update the PR.

**Comment by aesalazar** [comment]
Hi everyone.  I have not been keeping up.  Has dependency support been added?  I say the other some mention of what looked like it when running a `winget upgrade` the other day.  Any docs on it?

**Comment by Trenly** [comment]
> Hi everyone. I have not been keeping up. Has dependency support been added? I say the other some mention of what looked like it when running a `winget upgrade` the other day. Any docs on it?

Dependencies are supported in the 1.6 client, but we are still waiting for the changes to be rolled out fully

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> Faulting application name: LogitechOMM.exe, version: 1.1.6805.0, time stamp: 0x628bc94e 
> Faulting module name: KERNELBASE.dll, version: 10.0.19041.1202, time stamp: 0xc9db1934 Exception code: 0xe0434352 Fault offset: 0x0000000000034f99 
> Faulting process id: 0x1498 
> Faulting application start time: 0x01da34f691621602 
> Faulting application path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Logitech.OnboardMemoryManager__DefaultSource\LogitechOMM.exe 
> Faulting module path: C:\Windows\System32\KERNELBASE.dll Report Id: d1127ec0-e738-4fe3-be7f-4ae75ef34ade 
> Faulting package full name:  
> Faulting package-relative application ID: 

(Automated response - build 487.)

**Comment by stephengillie** [comment]
Hi @aesalazar,

With the dependency, this package installs normally but doesn't run when launched, instead writing 2 errors to the Application log. One of those is included above. Is this expected - should these be launched in a different way?

**Comment by aesalazar** [comment]
@stephengillie Ya, thats the problem.  It needs the VC++ runtime installed.  But since dependencies are not supported yet this package is kind of stuck.

Ernie

> FYI, the dependency seems to be on the VC++ Runtime. I test in a sandbox with:
> 
> `winget install Microsoft.VCRedist.2015+.x64`
> 
> and it worked like charm. When support it added, I will update the PR.

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> Faulting application name: LogitechOMM.exe, version: 1.1.6805.0, time stamp: 0x628bc94e 
> Faulting module name: KERNELBASE.dll, version: 10.0.19041.1202, time stamp: 0xc9db1934 Exception code: 0xe0434352 Fault offset: 0x0000000000034f99 
> Faulting process id: 0x1af8 
> Faulting application start time: 0x01da390b181befed 
> Faulting application path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Logitech.OnboardMemoryManager__DefaultSource\LogitechOMM.exe 
> Faulting module path: C:\Windows\System32\KERNELBASE.dll Report Id: 17775a2c-b4ed-4d69-a139-c090ed1704c1 
> Faulting package full name:  
> Faulting package-relative application ID:   Application: LogitechOMM.exe Framework Version: v4.0.30319 Description: The process was terminated due to an unhandled exception. Exception Info: System.BadImageFormatException    at System.Reflection.RuntimeAssembly._nLoad(System.Reflection.AssemblyName, System.String, System.Security.Policy.Evidence, System.Reflection.RuntimeAssembly, System.Threading.StackCrawlMark ByRef, IntPtr, Boolean, Boolean, Boolean)    at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(System.Reflection.AssemblyName, System.Security.Policy.Evidence, System.Reflection.RuntimeAssembly, System.Threading.StackCrawlMark ByRef, IntPtr, Boolean, Boolean, Boolean)    at System.Reflection.RuntimeAssembly.InternalLoadFrom(System.String, System.Security.Policy.Evidence, Byte[], System.Configuration.Assemblies.AssemblyHashAlgorithm, Boolean, Boolean, System.Threading.StackCrawlMark ByRef)    at System.Reflection.Assembly.LoadFrom(System.String)    at OnboardMemoryManager.Helpers.AssemblyLoader.CurrentDomain_AssemblyResolve(System.Object, System.ResolveEventArgs)    at System.AppDomain.OnAssemblyResolveEvent(System.Reflection.RuntimeAssembly, System.String)  Exception Info: System.BadImageFormatException    at OnboardMemoryManager.App..ctor()    at OnboardMemoryManager.App.Main()    at OnboardMemoryManager.Program.Main(System.String[]) 

(Automated response - build 489.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> Faulting application name: LogitechOMM.exe, version: 1.1.6805.0, time stamp: 0x628bc94e 
> Faulting module name: KERNELBASE.dll, version: 10.0.19041.1202, time stamp: 0xc9db1934 Exception code: 0xe0434352 Fault offset: 0x0000000000034f99 
> Faulting process id: 0x10cc 
> Faulting application start time: 0x01da3a8a543a5954 
> Faulting application path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Logitech.OnboardMemoryManager__DefaultSource\LogitechOMM.exe 
> Faulting module path: C:\Windows\System32\KERNELBASE.dll Report Id: 4540d643-1647-47f8-8815-7ec13672135b 
> Faulting package full name:  
> Faulting package-relative application ID: 

(Automated response - build 489.)

**Comment by stephengillie** [comment]
> @stephengillie Ya, thats the problem. It needs the VC++ runtime installed. But since dependencies are not supported yet this package is kind of stuck.
> 
> FYI, I am doing all of this in the Windows Sandbox. I manually run the below winget command and then run OMM and it works find for me. Let me know if you are seeing something different.
> 
> Ernie
> 
> > FYI, the dependency seems to be on the VC++ Runtime. I test in a sandbox with:
> > `winget install Microsoft.VCRedist.2015+.x64`
> > and it worked like charm. When support it added, I will update the PR.

Yes, with the dependency included, this package installs and launches normally.

File: manifests/l/Logitech/OnboardMemoryManager/1.1.6805.0/Logitech.OnboardMemoryManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Logitech.OnboardMemoryManager
+PackageVersion: 1.1.6805.0
```
Comment (stephengillie) [inline manifests/l/Logitech/OnboardMemoryManager/1.1.6805.0/Logitech.OnboardMemoryManager.installer.yaml:5]:
```suggestion
PackageVersion: 1.1.6805.0
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by aesalazar** [comment]
@stephengillie Thanks for the commit!  

But I am now confused.  I thought we had to use (and wait for) the 1.6 spec for that to work?  I tested locally and it works even under the 1.4 version.  Am I missing something? 

Thanks again.

**Comment by stephengillie** [comment]
This manifest contains a term that is blocked:

> LicenseUrl: https://www.logitech.com/en-us/legal/eula.html

(Automated response - build 513.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**